source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -79,250 +72,104 @@ msgid "IoT agents: agents for different protocols."
 msgstr "IoT agents: agentes para diferentes protocolos."
 
 #: ../../source/internal-communication.rst:59
-msgid ""
-"Each service will be briefly described in this page. More information can be "
-"found in each component documentation."
-msgstr ""
-"Cada serviço será descrito brevemente nesta página. Mais informações podem ser "
-"encontradas na documentação de cada componente."
+msgid "Each service will be briefly described in this page. More information can be found in each component documentation."
+msgstr "Cada serviço será descrito brevemente nesta página. Mais informações podem ser encontradas na documentação de cada componente."
```

Comment:
Why have line breaks been removed in some many places?

Reply:
Did you change it manually?

Reply:
When I run sphinx-intl it update the .po files, some changes are made by the "plugin"

Reply:
When I fix some items with poedit (accepting its suggestion) its also changes the fille.

Reply:
But it never happened to me that line breaks were removed